mParticle-Apple-SDK/Consent/MPCCPAConsent.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  MPCCPAConsent.swift
+//  mParticle-Apple-SDK
+//
+//  Created by Brandon Stalnaker on 9/17/24.
+//
+
+import Foundation
+
+/**
+ * Record of consent under the CCPA.
+ */
+@objc public final class MPCCPAConsent: NSObject, NSCopying {
+    
+    /**
+    * Whether the user consented to data collection
+    This should be set to false if the user has opted out of data sharing under the CCPA.
+    */
+    @objc public var consented: Bool = false
```

Comment:
```suggestion
    @objc public var consented = false
```
I'll add this to the style guide, but when inline assigning a value, no need to declare the type as it's inferred.